nubis/puppet/apache.pp

```diff
@@ -81,6 +81,13 @@
       'Remote_Addr 127\.0\.0\.1 internal',
       'Remote_Addr ^10\. internal',
     ],
+
+    # Create 2 canonical outging HTTP headers with Username and the groups they belong to
+    request_headers    => [
+      'set X-Nubis-SSO-Username "%{OIDC_CLAIM_email}e"',
```

Comment:
Is the `e` at the end intentional?

Reply:
Yes, it is! In Apache-land, the syntax to get at an environment variable in general is %{VARIABLE_NAME}e